README.md

```diff
@@ -221,6 +221,20 @@ const tokens = await oauth2Client.refreshAccessToken();
 // store these new tokens in a safe place (e.g. database)
 ```
 
+##### Checking access_token information
```

Comment:
Nit: Codify `access_token`? Here and below

Reply:
Fixed!